frictionless/project/interfaces.py

```diff
@@ -11,6 +11,7 @@
 class IFileItem(TypedDict, total=False):
     path: Required[str]
     type: str
+    errorCount: int
```

Comment:
@shashigharti 
I removed here `Optional` as `total=False` in `TypedDict` means everything is optional unless it's marked as `Required` - https://peps.python.org/pep-0655/

Reply:
@shashigharti 
And yea actually, it led to updating the actual logic as for `file.errorCount` we should have 3 states:
- undefined
- 0 (valid)
- X (invalid)
and we can't have `None` here

Reply:
Thank you!. I was making it complex :(

Reply:
Not at all it's just very confusing topic (missing values) especially when we need to pair Python and JavaScript